haddock-api/src/Haddock.hs

```diff
@@ -288,6 +289,23 @@ render dflags flags qual ifaces installedIfaces extSrcMap = do
 
     sourceUrls' = (srcBase, srcModule', pkgSrcMap', pkgSrcLMap')
 
+    -- TODO: This silently suppresses errors
```

Comment:
I am not sure, but it probably refers to the `(m, "") <- readP_to_S parseModuleId mod_str` further below. Because this pattern match occurs in a list comprehension, an parse errors translate into the reexported module just being dropped entirely from the list, no error.